samples/tfm/tfm_cert/child_image/mcuboot/prj.conf

```diff
@@ -0,0 +1,27 @@
+CONFIG_PM=n
+
+CONFIG_MAIN_STACK_SIZE=10240
+CONFIG_MBEDTLS_CFG_FILE="mcuboot-mbedtls-cfg.h"
+
+CONFIG_BOOT_SWAP_SAVE_ENCTLV=n
+CONFIG_BOOT_ENCRYPT_RSA=n
+CONFIG_BOOT_ENCRYPT_EC256=n
+CONFIG_BOOT_ENCRYPT_X25519=n
+
+CONFIG_BOOT_UPGRADE_ONLY=y # changed from default MCUboot
+CONFIG_BOOT_BOOTSTRAP=n
+CONFIG_MCUBOOT_DOWNGRADE_PREVENTION=y # changed from default MCUBoot
+
+CONFIG_FLASH=y
+CONFIG_FPROTECT=n # changed from default MCUBoot
```

Comment:
Isn't it desirable for MCUBoot to protect itself and its slots?

Reply:
I got this from hello_world sample, and I didn't investigate why.
Following the fix here: https://github.com/nrfconnect/sdk-nrf/pull/9206 this should now be removed.